### PR TITLE
Nerfs Autocloner

### PR DESCRIPTION
**Pull request by Poojawa**
:cl:
balance: Cloning pod no longer gives good or bad mutations
balance: Autoprocess no longer automatically clones people
/:cl:

Been a major brouhaha for a great number of years at this point. This simply removes mutations good and bad from clones and makes autoprocess only scan people, not autoclone. You will need to occasionally check records and on people in high risk jobs such as mining or Security.

### CODE REVIEW

**Comment by RogueSteampunker** [comment]
And once again, we fail to realize just why autoprocess existed in the first place, and that's cause medical never actually checks it.

In all seriousness, removing the autoclone function is a needless nerf that does nothing but ensure you're overtly likely to never get cloned even if you prescanned, cause the only two doctors on the ship either cryoed, went off to fuck, or just don't care. This has been documented dozens of times, and no amount of drilling into their brains has fixed this.

Besides that though, removing mutations from it in general is alright

**Comment by Poojawa** [comment]
it solves both issues just fine. Pointing out that no-one works medbay as a reason is rather amusing, considering that if medical had things to do and people to treat, more people would be interested in maintaining it. 

With so many outside ways of 0 risk healing, why should anyone sign up for it? Just go to botany, or the bartender, or ask cargo for First Aid kits. People just break into chemistry to steal the dispenser for mutagen for botany, then no-one needs to use chemistry unless they're making bombs.

**Comment by Ludox235** [comment]
I DIDNT KNOW THE ERP CORRUPTED THE HALF OF THE BRAIN THAT IS LEFT FOR YOU TO USE.
AT THE VERY LEAST ROGUESTEAMPUNKER KNOWS.
LITERALLY, NO FUCKING BODY CHECKS THE CLONER. THIS IS WHY NOBODY GOES TO GET PRESCANNED IF AUTOCLONING ISN'T ON, THAT'S BECAUSE MOST OF THE TIME NOBODY IS THERE TO SCAN YOU, AND IF YOU GET SCANNED ANYWAY YOU WON'T GET CLONED BECAUSE NOBODY CHECKS THE CLONER. IF ON NORMAL SERVERS NO ONE CHECKS THE CLONER THEN ON A SERVER WHERE BORED PEOPLE CAN FUCK EACH OTHER IN A GENITAL WORSHIP IN DORMS IT'S GONNA BE WORSE. AT THE VERY LEAST ONCE SOMEONE DIES AND IS PRE-SCANNED MAKE THE CLONER SAY IN MEDICAL RADIO: "Cloning machine (machine): Someone has died."

**FUCKING CAPS LOCK HOURS.**

**Comment by nik707** [comment]
I'm pretty hard against this for what's already been said, so I won't restate it.

**Comment by RogueSteampunker** [comment]
>it solves both issues just fine. Pointing out that no-one works medbay as a reason is rather amusing, considering that if medical had things to do and people to treat, more people would be interested in maintaining it.

And I'll say it again, making it so it's a constant chore doesn't make people do it. I've literally had med doctors complaining they have nothing to do, when they don't even check cloning until an admin literally has to subtle them to go give it a look. And once again, making medical interesting and worthwhile to play and flooding it with people screaming for help are two different things.

>With so many outside ways of 0 risk healing, why should anyone sign up for it? Just go to botany, or the bartender, or ask cargo for First Aid kits. People just break into chemistry to steal the dispenser for mutagen for botany, then no-one needs to use chemistry unless they're making bombs.

Then why not tone back basic chem healing, make higher-tier chems actually worth it, and replace one of the sleepers with a stasis sleeper so it's more reasonable to do massive undertakings to heal someone. Even making defib timers longer and teaching people about the wonders of strange reagent would go a long way in making the job more engaging without adding actual content, and all this does is make more medical doctors quit when they're screamed at for keeping people dead for an hour and a half.

**Comment by Poojawa** [comment]
Ghetto chemistry and botany outstrip any reason to do inhouse chemistry except for quick bomb making.

Defib timers being longer tends to shift defibs as the sole instance of revival than cloning because it's completely instant in its revival. Just as in the case of when sleeper units treated people down to -80 health that no-one used the time consuming cryo. Forcing people to make do with alternative cloning things isn't even in the scope of what this changes. It's not a compromise, just some measure of a separate change.

Doctors who don't do their job deserve to be yelled at. It's no different than Security, or Science, or Engineering, or the Captain, or Mining, or Cargo when they're slouching off.

They already get screamed at because no-one turns autoprocess on. So nothing really changes.

It really isn't that difficult to check the crew monitor, and ask mining if they're okay. If they see something weird and someone actually came in to get a scan, check the scanner. Ok hey, it's telling me they're dead -> clone starts. simple.

**Comment by deathride58** [comment]
>removing bad mutations from cloning entirely
y tho

**Comment by RogueSteampunker** [comment]
>Ghetto chemistry and botany outstrip any reason to do inhouse chemistry except for quick bomb making.

Ghetto chem is tedious and rough to mass produce, and expecting botany to actually do anything helpful consistently is laughable. It will always be better to just go check the chem fridge, and claiming otherwise makes one have to wonder how much you actually play to come to that conclusion.

>Defib timers being longer tends to shift defibs as the sole instance of revival than cloning because it's completely instant in its revival. Just as in the case of when sleeper units treated people down to -80 health that no-one used the time consuming cryo. Forcing people to make do with alternative cloning things isn't even in the scope of what this changes. It's not a compromise, just some measure of a separate change.

...except the fact that cloners won't heal above 20 unless they're upgraded, and having to heal someone vs. put them in the cloner should be encouraged if you actually want medical to be doing something.

>Doctors who don't do their job deserve to be yelled at. It's no different than Security, or Science, or Engineering, or the Captain, or Mining, or Cargo when they're slouching off.

So instead of leaving cloning as is in it's inoffensive state, we just discourage people from wanting to play medical by once again forcing admins to have to reprimand them. Fun.

>They already get screamed at because no-one turns autoprocess on. So nothing really changes.

It does change cause most scientists will just turn it on themselves. The case where medical didn't notice it only further proves the point.

>It really isn't that difficult to check the crew monitor, and ask mining if they're okay. If they see something weird and someone actually came in to get a scan, check the scanner. Ok hey, it's telling me they're dead -> clone starts. simple.

And yet none of medical does this. At all. Not even when the cloner remains unupgraded. Funny how that one works.

**Comment by nik707** [comment]
Adding more tedium is 100% not how to get more people to play. Having to babysit cloning and check it every few minutes is 100% just tedium. This is a video game, people want to have fun. What isn't fun is dying to something and losing your round because medical is A) fucking or B) not checking because "lol someone elses problem"

**Comment by Poojawa** [comment]
> y tho

Because the code is inconsistent with the random variable. It's a holdover from when limbs didn't start removed for speed cloning. 

>Ghetto chem is tedious and rough to mass produce, and expecting botany to actually do anything helpful consistently is laughable. It will always be better to just go check the chem fridge, 

Unless there isn't anyone working medbay and chemistry is already ransacked for the chemical dispenser. Then who is going to stock the firdge?

>...except the fact that cloners won't heal above 20 unless they're upgraded, and having to heal someone vs. put them in the cloner should be encouraged if you actually want medical to be doing something.

None of these changes effect cloning effectiveness except in genetic dice roll to randomly be empowered or require additional chemistry. Which, as you've pointed out, is a rare enough instance as is. 

>So instead of leaving cloning as is in it's inoffensive state, we just discourage people from wanting to play medical by once again forcing admins to have to reprimand them. Fun.

None of these changes alters current play requirements, Someone not doing the job shouldn't be mitigated just because. It's like asking for a public armory to fight antags because no-one signed up for Security. Your point isn't in the scope of these changes.

>It does change cause most scientists will just turn it on themselves. The case where medical didn't notice it only further proves the point.

Scientists often don't walk into the Clone lab with the help of a bluespace RPED. Which was added because scientists were breaking into the cloning lab to upgrade parts and it saved them the hassle. 

>And yet none of medical does this. At all. Not even when the cloner remains unupgraded. Funny how that one works.

That's a player issue that is easily corrected if people cared about the job they signed up for. Also this comment is outside the scope of this PR.

**Comment by RogueSteampunker** [comment]
itt: Pooj shuts down any consideration for the lack of capable medical staff by saying "outside the scope of this PR" constantly.

>None of these changes effect cloning effectiveness except in genetic dice roll to randomly be empowered or require additional chemistry. Which, as you've pointed out, is a rare enough instance as is.

I meant sleepers, was a mistype on my end

>Scientists often don't walk into the Clone lab with the help of a bluespace RPED. Which was added because scientists were breaking into the cloning lab to upgrade parts and it saved them the hassle.

which means either there's no med staff to let them in, or medical doesn't exist. funny how that one works.

>That's a player issue that is easily corrected if people cared about the job they signed up for. Also this comment is outside the scope of this PR.

yes, because believing your PRs exist in a vacuum makes them so much easier to pass through rather than considering the people this will be aimed at, bonus points for effectively saying "Let the admins take care of it."

**Comment by CameronWoof** [comment]
My main beef with this is how tedious and unfun it would be to play as Medical and have to periodically go down the list of records, trying to clone each one until it turns out someone was dead.

A "scan all" button or something that highlights which entries on the list are cloneable would go a long way to remedy this.

**Comment by Poojawa** [comment]
>itt: Pooj shuts down any consideration for the lack of capable medical staff by saying "outside the scope of this PR" constantly.

and there went any further discussions with you in good faith. 

>yes, because believing your PRs exist in a vacuum makes them so much easier to pass through rather than considering the people this will be aimed at, bonus points for effectively saying "Let the admins take care of it."

same as with this. Not once have I said this is an admin issue to resolve. Kindly stick to the topic without making direct remarks about myself and to what you believe my intentions are.

**Comment by Poojawa** [comment]
>A "scan all" button or something that highlights which entries on the list are cloneable would go a long way to remedy this.

This would be an excellent QoL improvement. although I'm not sure how to implement it properly with Tgui fuckery it would require

**Comment by Ludox235** [comment]
TL;DR POOJ THINKS THAT MAKING PEOPLE CHECK CLONING EVERY 10 MINUTES WILL FORCE THEM TO BE "INTERESTED" IN THEIR JOB. ALSO SUIT SENSORS DON'T GO THROUGH Z-LEVELS, ALSO, YOU CANNOT FORCE PEOPLE TO DO SOMETHING BY CHANGING STUFF. AUTO-CLONING IS FINE, YOU'RE NOT NERFING SOMETHING, YOU'RE NOT MAKING THE GAME ANY BETTER IN ANY WAY, YOU'RE JUST GIVING MEDICAL A NEW THING TO DO, EVERY 10 MINUTES. TELL ME, HOW IS IT FUN IN ANY WAY TO HAVE TO CLICK 10 BUTTONS EVERY 10 MINUTES TO SEE IF SOMEONE DIED OR NOT?

**Comment by RogueSteampunker** [comment]
>same as with this. Not once have I said this is an admin issue to resolve. Kindly stick to the topic without making direct remarks about myself and to what you believe my intentions are.

and yet

>Doctors who don't do their job deserve to be yelled at. It's no different than Security, or Science, or Engineering, or the Captain, or Mining, or Cargo when they're slouching off.

and 

>That's a player issue that is easily corrected if people cared about the job they signed up for. Also this comment is outside the scope of this PR.

both seem to imply that you're effectively asking for admins to have to cull any and all medical doctors who don't adhere to the slog of checking cloning every ten minutes. Unless you're suddenly expecting people to stop being shit all the time through IC means, in which case good luck.

>nd there went any further discussions with you in good faith.

you literally shut down half of my points with "it's outside the scope of this pr."

**Comment by Poojawa** [comment]
@Ludox235 
capslock

**Comment by CameronWoof** [comment]
I don't know about the PR in its current state. I understand that what I suggested doubles or triples the difficulty of the PR, but taking a powerful tool and replacing it with an unfun activity that, it seems to me, people would be unlikely to subject themselves to doesn't seem like a step forward.

SS13 is a game where you can do everything "right" and still die to something unexpected, and increasing the likelihood that your recovery from that state hinges on historically uninterested players doing something that isn't fun seems like it's going to increase frustration.

**Comment by Poojawa** [comment]
It's not been replaced? The thing people aren't realizing is that the autoprocess upgrade is an upgraded feature. Notably a very controversial one at that. Removing cloning is far too harsh. Removing prescans is far too harsh. This maintains functionality while curbing laziness being rewarded. No one likes lazy doctors. 

It has never been something that the cloner starts off with either. Sure upgrades tend to be quick enough that it sure feels like a default feature, but everyone experienced with the game already is well aware of it not coming as a round start package. Good medical doctors are already in the habit of checking on the crew. the one task they have is to ensure others are hale and hearty. That's literally all they have to do as a responsibility. They obviously are taken for granted as well, judging by how the majority of counter argument is based upon likely new players to medical who aren't otherwise in the habit or haven't been instructed on why it's a habit.

Death alarms have been suggested previously as a means to alert med staff, but with so many ways of being able to listen in on the medical channel, that murderous conflict is a cornerstone of this server's game play; it would have put too much meta info into a skilled player's hands. This is also the reasoning of why removal of specific names was done from the announcement radio.

**Comment by nik707** [comment]
Okay so your way to fix lazy doctors is actually to punish anyone who has to rely on them which is some pretty twisted logic and honestly won't do anything. We already struggle with people not doing their job, especially on main. Making it *more* annoying for everyone *else* isn't the way to fix this.

**Comment by Poojawa** [comment]
Then how do you suggest fixing the issue of lazy doctors?

**Comment by RogueSteampunker** [comment]
by making medical actually interesting and treating it like the rest of the departments, rather than believing that making the job more monotonous makes it interesting. Making limbs able to be swapped about much more freely, giving them easier access to implants and better organs like robotics does. This way, the only people being lazy are the ones who're just trying to goof off all shift and can be reprimanded without much loss, rather than the ones who just don't want to have to check cloning every ten minutes cause a coder who hasn't touched the job in at least weeks says it's easy.

**Comment by nik707** [comment]
Pretty much what steampunker said, and pretty much anything that isn't "introduce more tedium" like most of your answers seem to be when it comes to game/job balance.

**Comment by Poojawa** [comment]
How do you suggest to make it more interesting that simultaneously isn't tedious to do round after round?

What sorts of implants are good enough to be made accessible easier?

Why shouldn't these implants essentially just be round start if they're useful/needed?

Do they require surgery to implant? Or will it just quickly boil down to making it a protolathe thing for a reusable, self serve auto doctor?

What limbs are needing required changes to swap? With my marking changes, actually putting different limbs on people makes them look different, by the by.

How easy should limb swapping be? Is it still so easy that robotics can still do it and just as fast if not because they're the ones with access cyborg limbs? Is it easy/accessible enough that it gets used by ghetto surgery?

What advantages would swapping limbs with a monkey give? How would grafting limbs of a creature made of acid for blood not murder you in moments?

How will limb implantation and implant giving encourage people to play the job more?

**Comment by RogueSteampunker** [comment]
>What sorts of implants are good enough to be made accessible easier?

I mean, to start with, most the implants that robotics gets access to would be a nice change of pace, considering they already get access to most of science, mechs, cyborgs, and the like that having to share surgery duties with medical wouldn't be that big of a deal.

>Why shouldn't these implants essentially just be round start if they're useful/needed?

That's like asking why botany doesn't get a full dna machine round start, or why the xenobio consoles don't have all their disks at roundstart.

>Do they require surgery to implant? Or will it just quickly boil down to making it a protolathe thing for a reusable, self serve auto doctor?

I mean, surgery would be a must, yes. We're not just gonna throw some implants at them and hope it works with an auto-implanter.

>What limbs are needing required changes to swap? With my marking changes, actually putting different limbs on people makes them look different, by the by.

I mean, mostly just looking into making it so limbs aren't specifically held to a certain part of the body, or that a carbon is limited solely by the amount of limbs they start with. I know there's code support for multiple arms as shown by tg's vving of them and their wheelchairs considering additional arms, so adding potential drawbacks and benefits to additional body modifications could make medical a far more visited area for more than just wounds. Effectively, just lifting the limitation that "x limb can only go in y" bring, even if that just means getting to replace someone's head with a monkey's.

>How easy should limb swapping be? Is it still so easy that robotics can still do it and just as fast if not because they're the ones with access cyborg limbs? Is it easy/accessible enough that it gets used by ghetto surgery?

I mean, limb swapping as is shouldn't really be changed much, since it works well as is. However, if the frankensteining concept a fair few have talked about would get considered, a drawback like potential infections or chances of limb malfunctions would need to be present lest medical just turn into a powergame facility of ten-armed freaks.

>What advantages would swapping limbs with a monkey give? How would grafting limbs of a creature made of acid for blood not murder you in moments?

I mean, more than likely just for the joke of it or a gimmick. Afterall, there's no reason to make a slime corgi in xenobio, but people just do it anyways cause it's funny or interesting. Just the same that letting someone potentially turn into the monkey-headed freak who prowls maintence could be just as interesting. And as for stuff like acid, maybe it will just melt into you, hence making the need for constant burn medications with the potential plus of turning half-xeno rather worthwhile, if not extremely painful.

>How will limb implantation and implant giving encourage people to play the job more?

Because quite simply, it gives something that isn't an inherent chore for medical to do, that still has a genuine desire from the crew to want. It gives medical the ability to do gimmicks that aren't just "stutter slut or drunkard that steals your organs", while also giving them the potential to try and experiment with just how far they can push the human body, to see just what they can make their body contort into. Or you could totally just say that giving someone arms for legs while you used strange reagent on them was completely necessary. In the end, it's just the start that helps push towards giving medical stuff that ISN'T just more of a job that they never asked for.

**Comment by Poojawa** [comment]
So essentially, what you want is more gimmicks being a possibility, since not many, if any people are going to take up a doctor slot with their weird Transformation fetish partner  and just graft limbs on to them round after round. Consider the time scale of 100 rounds in the future for people doing this gimmick. 

I feel like giving wider access to nodrop and auto-heal, xray, thermal implants would get abused as heavily as the concept of nanites is. People wouldn't be coming into medbay to be treated by a doctor, they'll just break in and print themselves off implants. Especially if self-surgery was re-introduced.

Not to mention taking a science based thing and overriding the medical equivalents of virology, genetics, and even the doctor's job itself would further dilute the interest in playing those parts of a department. Why bother sitting in viro making a heal virus when you can just print it from the protolathe after half an hour?

Why is the concept of a job a chore? is there something inherently bad with having tasks to maintain while performing a job?

Patrolling the station is a duty of Security, it's a chore.
But that's the job. 

Setting up the engine and patching the odd window to space from dust is a chore.
But that's the job of Engineering.

Why is medical having to occasionally check a cloner suddenly inexcusable amounts of tedium that should never be inflicted upon people. It's part of their duties. It's the job. How is it requiring them to do more than they asked for, when it's part of the job they signed up for?

**Comment by nik707** [comment]
Because fixing windows comes with an inherent danger, and can lead to other events. Patrolling comes with an inherent danger, and the dynamic of the station. Walking to cloning and checklisting people for cloning comes with literally nothing, and is more akin to data entry than gameplay.

**Comment by RogueSteampunker** [comment]
>So essentially, what you want is more gimmicks being a possibility, since not many, if any people are going to take up a doctor slot with their weird Transformation fetish partner and just graft limbs on to them round after round. Consider the time scale of 100 rounds in the future for people doing this gimmick.

I mean, it adds a great deal besides just being a gimmick. It makes it so that medical can actively make an impact on others, with potential limitations and maintences needed to ensure that it only gets done sparingly. And if anything else, it'd be welcomed content in a department that has none.

>I feel like giving wider access to nodrop and auto-heal, xray, thermal implants would get abused as heavily as the concept of nanites is. People wouldn't be coming into medbay to be treated by a doctor, they'll just break in and print themselves off implants. Especially if self-surgery was re-introduced.

Except the fact that robotics already gets access to all of this stuff. What would change besides giving medical a chance to pimp people out instead of robotics?

>Not to mention taking a science based thing and overriding the medical equivalents of virology, genetics, and even the doctor's job itself would further dilute the interest in playing those parts of a department. Why bother sitting in viro making a heal virus when you can just print it from the protolathe after half an hour?

...excuse me? How is any of that an issue when we're talking about the base implants that already exist, and would simply be either added to the medical prolathe, or moved from the robotics prolathe to medical? And by all means, viro has needed a serious rework for a long time, so trying to limit major changes because it's still nearly useless is a silly move.

>Why is the concept of a job a chore? is there something inherently bad with having tasks to maintain while performing a job?

I mean, the fact of the matter is that most job's chores tend to have some level of stimuli, such as janitor, bartender or chef, or offer rewards or events for doing them. Engineering gets fancy tools and some of the best access for building and rennovation, alongside full on space access. Security gets the entirety of the weapons branch of RnD and are the forefront of station-based action against groups like cults another otherwise. Science gets the entirety of xenobio, nanites, circuits, and the like.

And what do you get from doing medical? Nothing. At best, you get a few prolathe tools that do little but speed up bits like surgery and otherwise when those are the parts you wouldn't want to finish up so quickly. And as we've seen from constant rounds, simply healing people isn't near as satisfying as cleaning the damn floors.

Quite simply, no one just does their job for the sake of maintaining it. Even with the janitor there's the catharsis of the work and getting to know that your work actively helps the mood of the station, whereas medical is thankless and gives little reason to want to do more than the bare minimum cause you won't get anything for it.

>Patrolling the station is a duty of Security, it's a chore.
But that's the job.

And in exchange they find crimes, potential antags, and get fancy guns.

>Setting up the engine and patching the odd window to space from dust is a chore.
But that's the job of Engineering.

And in exchange you get enough tools and access to make even the most hardened of greytider's blush.

>Why is medical having to occasionally check a cloner suddenly inexcusable amounts of tedium that should never be inflicted upon people. It's part of their duties. It's the job. How is it requiring them to do more than they asked for, when it's part of the job they signed up for?

Because there's no reason to force that into their job if it'll do nothing but make the job even less desirable. After all, can you even justify walking away now if someone might need cloned at a moment's notice cause you have to manually do it, and with little way to know someone needs to be cloned outside of word of mouth, sensors (if you're lucky) or checking it every couple of minutes? And who wants to have to deal with the tgui as they click through six or more records?

Congrats, Pooj, you'll have given medical a potential reason to get bwoinked if there's even one record in the cloner. I'm sure that'll make people run to play medical in droves.

**Comment by silicons** [comment]
My answer's no for the reasons everyone have already stated.
I'm alright with the removal of auto-superpowers though.

**Comment by ariever** [comment]
Last thing i need is a fnaf type 'check everything every 30 seconds' game or get yelled at for not doing my job icly and oocly.

Its something to do, ill buy that. But is it fun? Fuck to-the no.

In response to the PR itself, i dont think any medical person would enjoy it, myself included. 

Honestly pooj, I don't understand what planet you hailed from, but here on Earth we don't consider repediative, mundane actions 'fun' and 'engaging'. It makes me feel like I'm back at school, or like I'm working a 9-5 office job. Sure this game is about work, but it's fun work. Making things explode, cutting things out of people, operating heavy mechs, shooting guns. Pressing 10-20 buttons on a list doesn't sound like fun.

Whats the point of 'auto' cloning if its not cloning automatically? What's the point of a ground breaking research station if each PR you make turns the equipment and the station itself into more inferior versions of itself?

**Comment by ariever** [comment]
Oh i was late to the pr bashing party :c im watching tv srry

**Comment by Poojawa** [comment]
>I mean, it adds a great deal besides just being a gimmick. It makes it so that medical can actively make an impact on others, with potential limitations and maintences needed to ensure that it only gets done sparingly. And if anything else, it'd be welcomed content in a department that has none.

Such as? Also, how does medical have 0 content to do? Why is it completely and totally useless unless it has an autocloning set up?

Is cloning the only thing that makes medical worth anything? If so, why?

>Except the fact that robotics already gets access to all of this stuff. What would change besides giving medical a chance to pimp people out instead of robotics?

It's one less thing for a subdepartment that literally has nothing to do except the occasional borg and someone making a Phazon/Gyagax to fight antags or because they are an antag. 

>...excuse me? How is any of that an issue when we're talking about the base implants that already exist, and would simply be either added to the medical prolathe, or moved from the robotics prolathe to medical? And by all means, viro has needed a serious rework for a long time, so trying to limit major changes because it's still nearly useless is a silly move.

Literally what you're suggesting is to supplant genetics powers (Xray) and viro's helpful symptoms (auto healing, food supply) with stuff that can be easily printed off. Not only that, but as stated previously you're removing content from a subdepartment that has a very limited range of actual duties. No AI or Borgs? Literally nothing to fix/upgrade. No one wants a Mech. Can't... really build one then. What else is there for Robotics to do?

>Because there's no reason to force that into their job if it'll do nothing but make the job even less desirable. After all, can you even justify walking away now if someone might need cloned at a moment's notice cause you have to manually do it, and with little way to know someone needs to be cloned outside of word of mouth, sensors (if you're lucky) or checking it every couple of minutes? And who wants to have to deal with the tgui as they click through six or more records?

What the hell are you on about. Are you seriously arguing that a medical staffer shouldn't have to do medical work? No one comes pre-scanned - This ain't RP server with their sleeve machine. Since when does 'periodically checking' mean 'NEVER LEAVE THE CONSOLE EVER'? Can you please refrain from hyperbole?

>Congrats, Pooj, you'll have given medical a potential reason to get bwoinked if there's even one record in the cloner. I'm sure that'll make people run to play medical in droves.

It's... literally what's already been a thing my dude. There's no reason to making this out as if I'm removing cloning entirely.

**Comment by Knightfall5** [comment]
Periodically checking does mean sifting through ten to twenty scans during highpop and trying to clone every single one multiple times a shift. Oh boy that sure does sound like fun engaging gameplay that a doctor will totally do! Unless someone calls out they are dead, they will never get cloned. Period, end of story.

 If you want to nerf autocloners in regards to antags, just give it a three or four minute wait after death until they clone. This gives the antag time to mess up the cloner in a subtle way to insure they don't clone.

**Comment by Poojawa** [comment]
if only there was some form of device. Something perhaps handheld that you could use that monitors crew health. Perhaps said device might also help to cut down on excess checks to records?

Isn't there a time honored, repeated announcement of medical to turn a particular, clothing based sensor suite that tracks pertinent information of health?

**Comment by nik707** [comment]
Two maintainers have already turned down the autocloner change. No sense in continuing to argue. The nerf to powers is fine, alter the PR to do that or create another for it.

**Comment by Poojawa** [comment]
two halfassed responses, one involved just because it's me. the other just blandly agreeing that it somehow creates unreasonable expectations to do a simple task.

**Comment by RogueSteampunker** [comment]
Imagine literally just screaming suit sensors at the top of your lungs and expecting that to justify a shit change.

I'm just saying, pooj. When suit sensors don't work for miners and can get removed or just plainly ignored by most others, it's not an excuse to have to go through the cloning records every ten minutes. And when you can't walk away for more than a slight wank cause you don't wanna get screamed at, all this change does is make medical even more fucked.

You're genuinely confusing content and doing one's job with adding unnecessary hoops and don't need to exist for any reason besides making the medical staff his that hard *suicide

**Comment by RogueSteampunker** [comment]
*to not have to go through the records, rather. Mobile posting, I swear.

**Comment by Poojawa** [comment]
Imagine actually arguing that signing up for a job is the maximum amount of effort to be asked of someone signing up to play. 

Imagine having standards for roleplay gameplay so low that having your job done for you is preferable to doing your job.

Imagine confusing the basics of your job as unnecessary hoops to do your nondescript work or... something.

What is expected of a medical doctor anyway? I wanna know now, because apparently I'm just so inconceivably unable to grasp past the idea that a medical practitioner would be, to me at least, understandably encouraged to be proactive in applying medical treatment and services to the crew.

Please explain to me what a medical doctor's expected duties are.

and, if you're so very generous, explain why those duties are either unfulfilled or why the automation of duties is essential for their gameplay

**Comment by Darkwolf6180** [comment]
By what means do you expect fun to be derived?
Rather, this will result in entertainment deprived.
Autocloner is the greatest grace in medical,
As after all, half the department is mental.
With sliding slips your PR falls streaking,
Down, down, into the pit of reeking.

**Comment by CozmoB** [comment]
I'd suggest maybe adding more downsides to being cloned such as chances of missing limbs even at a full clone rather than requiring periodic checking of the autocloner.

**Review by nik707 — CHANGES_REQUESTED** [review]
Cut the autocloner nerf. I don't care why you think I'm blocking it. I've allowed plenty of your PRs through, including getting help you you in my custom suit. If you want to hide a bad idea behind a veil of "He just hates it cus it's me!" go for it. But it's still gotta be changed.

**Review by silicons — CHANGES_REQUESTED** [review]
No autocloner nerfs. Just remove the superpower on T4.
Keep the bad mutations below T3 too, those are fine.
